classes/Book/label.rb

```diff
@@ -0,0 +1,15 @@
+class Label
+  attr_accessor :items, :color, :title, :id
+
+  def initialize(color, title, id = nil)
+    @id = id || Random.rand(1..1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.label = self
```

Comment:
The requirements say:
> - add_item method in the Label class
>   - should add self as a property of the item object (by using the correct setter from the item object)

the `Label` uses the `add_item` method to set the `Item` instance into `items` collection, I think it would be better to have an `add_label` setter method in `Item` class in order to follow the requirements, and set the `label` property for `Item` as read only. Because you can do something like `item.label = 1` and `1` is not an `Item` instance.